README.md

```diff
@@ -72,6 +72,14 @@ hip_mandrill:
         sender: info@example.com
         sender_name: John Doe
         subaccount: Project # Optionally define a subaccount to use
+    proxy:
+        use: true # when you are behing a proxy. Default value is 'false'
+        host: example.com
+        port: 80
+        user: john
+        password: doe123
+    ssl:
+        verify: true # verify the SSL certificats. Default value is 'true'
```

Comment:
I think you are right: it's better to fix this issue by using a valid certificate instead of disabling the SSL check of curl.
I'll do that tomorrow!